### DISCUSSION TITLE
App crash if vue devtool open.

### DISCUSSION THREAD

**Discussion by juac08** [vuejs/devtools-v6#1722] [Q&A] [answered]
" Sub is not a function" Nuxt app. In dev

**Comment by Akryum** [comment, marked as answer]
Go to Vue 2 plugin settings in the devtools and check "Legacy Actions"

![image](https://user-images.githubusercontent.com/2798204/153394990-62376d5d-b062-4f30-be0f-6354a05f25b4.png)

**Reply by juac08** [reply]
Thank you so much. Its working now. Legacy actions was disabled. I just enabled it and working fine.
